pandas/io/json/_json.py

```diff
@@ -52,8 +53,11 @@ def to_json(
         )
 
     if path_or_buf is not None:
-        path_or_buf, _, _, _ = get_filepath_or_buffer(
-            path_or_buf, compression=compression, mode="w"
+        path_or_buf, _, _, should_close = get_filepath_or_buffer(
+            path_or_buf,
+            compression=compression,
+            mode="w",
```

Comment:
wt?

Reply:
I suppose? I didn't write this, it's due to style reformatting.

Reply:
Done